solidity/ecdsa/contracts/libraries/DKG.sol

```diff
@@ -33,6 +34,8 @@ library DKG {
     using ECDSA for bytes32;
 
     struct Parameters {
+        // Time in blocks during which a seed is expected to be delivered.
+        uint256 seedTimeout;
```

Comment:
We need to make it crystal clear in documentation that the time when we await for a seed is not included as a time of DKG because DKG has not yet started - it needs a seed to start - and the DKG timeout does _not_ include the time the contract waits for the seed.